Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt

```diff
@@ -198,72 +211,106 @@ fun HomeContent(
                 }
             }
 
-            Column(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .verticalGradientScrim(
-                        color = MaterialTheme.colors.primary.copy(alpha = 0.38f),
-                        startYPercentage = 1f,
-                        endYPercentage = 0f
-                    )
-            ) {
-                // Draw a scrim over the status bar which matches the app bar
-                Spacer(
-                    Modifier
-                        .background(appBarColor)
-                        .fillMaxWidth()
-                        .windowInsetsTopHeight(WindowInsets.statusBars)
-                )
+            val scrimColor = MaterialTheme.colors.primary.copy(alpha = 0.38f)
+            Scaffold(
```

Comment:
Do we even need the Scaffold? Looks like it is sitting in a Column already that maybe could just be used seeing as its just a TopAppBar and then LazyColumn

Reply:
I was thinking a Scaffold would be good if we want the top app bar to dismiss while scrolling. Since I haven't implemented that yet (requires M3) I'll remove the Scaffold for now 👍